pom.xml

```diff
@@ -0,0 +1,5 @@
+<dependency>
```

Comment:
I'm not sure this partial pom.xml is useful, until the file is complete enough to compile. Mathew has another pr to do that. You'll probably need to merge after that's checked in

Reply:
This pom.xml isn't useful at all. I will merge with Matthew's pom.xml.